### ISSUE TITLE
Load data tutorial model fails to converge

### ISSUE THREAD

**Issue by edwardCao** [tensorflow/tensorflow#26463]
<em>Please make sure that this is a documentation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:doc_template</em>

**System information**
- TensorFlow version:
- Doc Link:

site/en/tutorials/load_data/images.ipynb

*Describe the documentation issue**
The model fails to converge most of times. I changed the steps per epoch to run through all the data but it fails to converge. A lot of times just stuck in high loss low accuracy. 

I noted last layer output logits. It this expected?

**We welcome contributions by users. Will you be able to update submit a PR (use the [doc style guide](https://www.tensorflow.org/community/documentation)) to fix the doc Issue?**

**Comment by edwardCao** [comment]
BTW I ran only on colab with GPU.

**Comment by jvishnuvardhan** [comment]
@edwardCao Could you point out which tutorial you are referring to? Please let us know If there is any mismatch in what was mentioned in the website and what you are seeing. Thanks!

**Comment by edwardCao** [comment]
site/en/tutorials/load_data/images.ipynb

**Comment by jvishnuvardhan** [comment]
@edwardCao Thanks for the link. I think intention of the tutorials is to teach how to load image dataset using tf.data. In order keep the tutorial simple as well as take minimum time, it was developed like that. There are other tutorials that talk about how to improve performance of a DL model. Thanks!

**Comment by jvishnuvardhan** [comment]
I am closing the issue. Thanks!
